packages/office-addin-sso/src/error-handler.ts

```diff
@@ -0,0 +1,45 @@
+import { showMessage } from "./message-helper";
+
+export function handleClientSideErrors(error: any): boolean {
+  let invokeFallBackDialog: boolean = false;
+  switch (error.code) {
+    case 13001:
+      // No one is signed into Office. If the add-in cannot be effectively used when no one
+      // is logged into Office, then the first call of getAccessToken should pass the
+      // `allowSignInPrompt: true` option. Since this sample does that, you should not see this error
```

Comment:
Need to remove the last sentence of this comment: "Since this sample does that, you should not see this error"

Reply:
fixed